.php_cs

```diff
@@ -37,7 +37,7 @@ return PhpCsFixer\Config::create()
             'php_unit_method_casing' => ['case' => 'snake_case'],
             'php_unit_test_class_requires_covers' => false,
             'phpdoc_types_order' => ['null_adjustment' => 'always_last'],
-            'simplified_null_return' => true,
```

Comment:
This had to change because https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2349 suddenly started showing its teeth [here](https://github.com/martin-georgiev/postgresql-for-doctrine/pull/66/files#diff-9c4a7b8971dca6f754bda9be34645333R50).